src/NuGet.Clients/VsExtension/NuGetPackage.cs

```diff
@@ -103,6 +105,13 @@ public sealed class NuGetPackage : Package, IVsPackageExtensionProvider, IVsPers
 
         public NuGetPackage()
         {
+            // Set IsDev14 in NuGet.Common.RunTimeHelper to pick %PROGRAMDATA%/NuGet/Config as the MachineWideBaseDir
+            // This is only needed for Dev14 VSIX
```

Comment:
Remove the comment.  A comment here shouldn't claim knowledge of how `RuntimeEnvironmentHelper.IsDev14` will be used elsewhere (e.g.:  other assemblies).